EventTicketingSystem.Tests/DataAccess/CityRepositoryTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using EventTicketingSystem.DataAccess.Interfaces;
+using EventTicketingSystem.DataAccess.Models.Context;
+using EventTicketingSystem.DataAccess.Models.Entities;
+using EventTicketingSystem.DataAccess.Services;
+using EventTicketingSystem.Tests.Helpers;
+using EventTicketingSystem.Tests.Seed;
+using FluentAssertions;
+
+namespace EventTicketingSystem.Tests.DataAccess
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.None)]
+    public class CityRepositoryTests
+    {
+        private readonly TestDataReader _dataProvider;
+        private ICityRepository _cityRepository;
+        private DatabaseContext _context;
+
+        public CityRepositoryTests()
+        {
+            _dataProvider = new TestDataReader();
+        }
+
+        [SetUp]
+        public async Task Setup()
+        {
+            _context = DatabaseHelper.CreateDbContext();
+            _cityRepository = new CityRepository(_context);
+            var seeder = new TestDataSeeder(_context, _dataProvider);
+            await seeder.SeedCities();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            _context?.Dispose();
+        }
+
+        [Test]
+        public async Task CityRepository_Find_ReturnsCity()
+        {
+            var city = await _cityRepository.Find(1);
+
+            city.Should().NotBeNull();
+            city.Id.Should().Be(1);
+            city.Name.Should().Be("Moscow");
+            city.Country.Should().Be("Russia");
+        }
+
+        [Test]
+        public async Task CityRepository_GetCities_ReturnsCities()
+        {
+            var expectedCities = await _dataProvider.GetCities();
+            var cities = (await _cityRepository.GetCities()).ToList();
+
+            cities.Should().NotBeNull();
+            cities.Should().HaveCount(expectedCities.Count);
+            for (int i = 0; i < cities.Count; i++)
```

Comment:
fluent assertion ShouldBeEquivalentTo will do all work from the line 56 to line 62

Reply:
Same for all places with more than 1 should. To remove Id props use FluentAssertion exclude. 
If you will create IEntity interface with an Id prop inly - you will have ability to create your own extension and hide excluding for all cases

Reply:
Fixed